app/assets/javascripts/config/angularRoutes.js.coffee

```diff
@@ -102,7 +102,7 @@
               deferred.resolve(ListingService.listing)
               if _.isEmpty(ListingService.listing)
                 # kick them out unless there's a real listing
-                return $state.go('dahlia.listings')
+                return $state.go('dahlia.listings-for-rent')
```

Comment:
Redirecting to the homepage makes sense!

Reply:
I don't understand. You don't want to access listings at all for now?

Reply:
in the case of a user going to a non-existant listing we want to redirect to the home page instead of the rental listings page. 

We could save it for later, but my guess it that we are likely to forget

Reply:
ok, on it

Reply:
done